### DISCUSSION TITLE
Interact with contract from frontend

### DISCUSSION THREAD

**Discussion by mgroovyank** [Cyfrin/foundry-full-course-cu#1884] [General]
I've completed Patrick's previous course which used Hardhat. In that course, he used ethers.js to interact with contract from frontend UI.
In the new course which uses foundry, Do we still use ethers to interact with contract from frontend UI?

**Comment by EngrPips** [comment]
Hello @mgroovyank, If I remember correctly, we still use `ether` to interact with contract from frontend UI in the `foundry-full-course` as well.
